lib/dash/dash_parser.js

```diff
@@ -217,8 +217,7 @@ shaka.dash.DashParser = class {
    * @exportInterface
    */
   onInitialVariantChosen(variant) {
-    if (this.manifest_ && this.lowLatencyMode_ &&
-        this.minTotalAvailabilityTimeOffset_) {
+    if (this.manifest_ && this.manifest_.presentationTimeline.isLive()) {
```

Comment:
You know, it occurs to me now that this function should probably have some comment explaining why it exists.

Reply:
Done!